### ISSUE TITLE
Practice exam 4.F order swap page 1

### ISSUE THREAD

**Issue by Tonylayton1** [Section508Coordinators/DHSA-TT-200-A#242]
Remove the randomized tab order swap from the header links. Remove r1e04 and r1e05 from the list of failures for 4.F. documentation is updated.

**Comment by mikebarklage** [comment]
Done.

**Comment by Tonylayton1** [comment]
verified on dev

**Comment by drewnielson** [comment]
Verified on dev
